_rules/meta-viewport-b4f0c3.md

```diff
@@ -24,22 +26,37 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
+This rule applies to each `content` attribute for which all of the following are true:
 
-- the `content` [attribute value][] has the `user-scalable` property; or
-- the `content` [attribute value][] has the `maximum-scale` property.
+- the attribute is defined on a `meta` element with a `name` [attribute value][] of `viewport`; and
+- the [attribute value][] has at least one of the `maximum-scale` or `user-scalable` keys.
+
+For the purpose of this rule, the [attribute value][] of the `content` attribute is a list of key/value pairs.
 
 ## Expectation 1
 
-For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `no`.
+For each test target's [attribute value][], at least one of the following is true:
+
+- `user-scalable` is not defined; or
```

Comment:
Good point.

Related to that, and given what we semi-often do, I wonder if it would be a good idea to allow expectations like:
> For each test target [which verify some condition], …

So that test targets that do not verify the condition would automatically pass that expectation.

In that case, that would allow to write:
> For each test target that defines `user-scalable`, …

And in general, anytime we have Applicability with an alternative, that would allow somewhat simpler expectation. And result in less rules overall, I feel these splittings are often diluting rules that should stay close together.

Alternatively, this could also be done with Composite rules. Having one Atomic rule for each property, and a Composite requesting both to not fail. It is a bit redundant, but avoids splitting rules that are really working together.

Reply:
Tracking the idea in https://github.com/w3c/wcag-act/issues/548